Misc/NEWS.d/next/Security/2019-11-18-16-17-56.bpo-38722.x3mECW.rst

```diff
@@ -0,0 +1 @@
+:mod:`runpy` now uses :meth:`io.open_code` which will trigger auditing events.
```

Comment:
`open()` and `io.open()` also trigger auditing events. And as far as I can tell, this won't change anything WRT auditing; it only allows setting an "open code hook" using [PyFile_SetOpenCodeHook](https://docs.python.org/3.8/c-api/file.html#c.PyFile_SetOpenCodeHook). So this text needs to be revised.

Reply:
I've updated with a better description of what the change really means/does.